include "--\n" as email signature, so kmail (maybe few others) kill rest
whenreplying to email. this effectively filters out junk when replying emails.

--- eventum-20060907/include/class.error_handler.php~	2006-09-07 23:29:47.528645595 +0300
+++ eventum-20060907/include/class.error_handler.php	2006-09-07 23:30:32.269649910 +0300
@@ -101,7 +101,7 @@
         }
         @$msg .= "That happened on page '" . $HTTP_SERVER_VARS["PHP_SELF"] . "' from IP Address '" . getenv("REMOTE_ADDR") . "' coming from the page (referrer) '" . getenv("HTTP_REFERER") . "'.\n\n";
         @$msg .= "The user agent given was '" . $HTTP_SERVER_VARS['HTTP_USER_AGENT'] . "'.\n\n";
-        $msg .= "Sincerely yours,\nAutomated Error_Handler Class";
+        $msg .= "-- \nSincerely yours,\nAutomated Error_Handler Class";
         // only try to include the backtrace if we are on PHP 4.3.0 or later
         if (version_compare(phpversion(), "4.3.0", ">=")) {
             $msg .= "\n\nA backtrace is available:\n\n";
